### PR TITLE
feature: added array/repeat.

### PR DESCRIPTION
**Pull request by diasbruno**

### CODE REVIEW

File: tests/spec/array/spec-repeat.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      expect(repeat(0, 'a')).toEqual([]);
+    });
+
+    it('should create an array of size 1 and fill with a value "a".', function(){
+      expect(repeat(1, 'a')).toEqual(['a']);
+    });
+
+    it('should create an array of size N and fill with a value "a".', function(){
+      expect(repeat(2, 'a')).toEqual(['a', 'a']);
+    });
```
Comment (roboshoes) [inline tests/spec/array/spec-repeat.js]: Would you mind also adding some tests with longer strings as well as numbers?

**Comment by roboshoes** [comment]
I think we just need to extend the test cases a bit further but other than that I think it's good.

**Comment by diasbruno** [comment]
Sure. I'll add both tests. Thanks, @roboshoes.

File: tests/spec/array/spec-repeat.js (hunk 1 above)
Comment (diasbruno) [reply tests/spec/array/spec-repeat.js]: Added a test for the longer string and using a number.

**Comment by roboshoes** [comment]
Thank you.

**Comment by roboshoes** [comment]
I've merged you're contribution and it's live on [1.2.2](https://www.npmjs.com/package/mout). Thanks!

**Pull request merged**